backtracepython/utils/__init__.py

```diff
@@ -0,0 +1,16 @@
+import json
+import logging
+import re
+
+def parse_json(data, log_warning=True):
```

Comment:
I find this name very misleading. `dump(s)` creates JSON, for parsing I'd expect to use `load(s)`

Reply:
I agree.